src/main/java/countdownlatch/B_przykładCountdownLatch/PodróżDoHogwardu.java

```diff
@@ -0,0 +1,46 @@
+package countdownlatch.B_przykładCountdownLatch;
+
+import fabryczkapomocnicza.MyThreadFactory;
+
+import java.util.concurrent.*;
+
+/**
+ * <p>Przykładowe zadanko - problem polega na tym, że mamy pociąg do Hogwartu ;D Wyobraźmy sobie, że kilku czarodziejów
+ * właśnie otrzymało list, więc lecą ile sił w nogach na pociąg.
+ * Konduktor jest na tyle spoko, że zaczeka na wszystkich, zanim ruszy.</p>
+ *
+ * @see Pociąg
+ *
+ * @author Kacper Staszek
+ * @author Marcin Ogorzałek
+ *
+ * TODO:Uruchom program kilka razy, prześledź działanie.
+ *  Zakomentuj oznaczoną linię w klasie Pociąg, uruchom program i zrozum działanie metody await();
+ */
+public class PodróżDoHogwardu {
+
+    private static final int ILOSC_ZAREZERWOWANYCH_BILETOW = 10;
+
+    public static void main(String[] args) throws InterruptedException {
+
+        BlockingQueue<Pasażer> pasażerowie = new LinkedBlockingQueue<>();
+
+        Pociąg pociąg = new Pociąg(ILOSC_ZAREZERWOWANYCH_BILETOW, pasażerowie);
+
+        CountDownLatch referencjaCounDownLatchZNaszegoPociągu = pociąg.getCountDownLatch();
+
+        ThreadFactory threadFactory = new MyThreadFactory("Pasażer");
+
+        //TODO: Zastanów się, czy egzekutor może mieć dowolny rozmiar?
```

Comment:
Jeśli czytam ten kod w myśl pierwszego danego mi zadania (zrozum kod) to czy to TODO wywłaszcza wątek czy nie? Oto jest pytanie. :-) Co autorzy by chcieli?